### ISSUE TITLE
`hub clone org/repo.wiki` fails with "Repository not found"

### ISSUE THREAD

**Issue by glittershark** [github/hub#1172]

**Comment by mislav** [comment]
Sorry for the late reply:

1. Does the `org/repo` repository exists?
2. Does it have wiki enabled?
3. You can enable debugging with `HUB_VERBOSE=1 hub clone org/repo.wiki`. Please do that and see if anything jumps at you. You can paste us the debug log, with parts of it anonymized if you wish.

**Comment by glittershark** [comment]
```
❯ HUB_VERBOSE=1 git clone [org redacted]/[repo redacted].wiki
$ git config alias.clone
$ git config hub.protocol
$ git config --bool hub.http-clone
> GET https://api.github.com/repos/[org redacted]/[repo redacted].wiki
> Authorization: token [REDACTED]
< HTTP 404
{"message":"Not Found","documentation_url":"https://developer.github.com/v3"}
$ git config hub.protocol
$ git config hub.protocol
$ git clone git://github.com/[org redacted]/[repo redacted].wiki.git
Cloning into 'SecurityScorecard-UI.wiki'...
fatal: remote error:
  Repository not found.
```

some more information:

- repo definitely exists
- repo wiki definitely exists
- repo is private
- it looks like since the github API request is returning a 404 it's trying to clone using `git://`, which obviously fails since it's a private repo

**Comment by glittershark** [comment]
Did some snooping - looks like https://github.com/github/hub/commit/93eccff35d245b3b9e86b802a51eea3d893255b4, which `master` has but `2.2-stable` doesn't, fixes the issue. Will leave closing this now vs at the next version release up to you

**Comment by mislav** [comment]
Thanks for all the info! You're right that that commit fixes it, and I'll port this to 2.2-stable to be in the next 2.2.4 release. Stay tuned!

**Comment by glittershark** [comment]
thanks @mislav!

**Comment by mislav** [comment]
Fix now included in the 2.2.4 release.
